tools/pv_tree_generator/build_tree.py

```diff
@@ -132,37 +137,45 @@ def build_tree(v, pop_obs_spec, stat_vars, show_all):
         'sv_set': set(),#used for counting child nodes
         'search_count': 0,
         'search_sv_set': set(),
+        'placeTypes': PLACETYPES,
     }
 
     # specs with 0 constaints are of type "value", 
     # as the level 1 children of root
     for pos in pop_obs_spec[0]:
         search_count = (1 if (pos.pop_type, pos.mprop, '') in SEARCH_SPECS else 0)
         ui_node = util.UiNode(pos, {}, False)
-        root['children'].append({
-            'populationType': ui_node.pop_type,
-            'show': 'yes',
-            'selected': 'no',
-            'expanded': 'no',
-            'argString': ui_node.arg_string,
-            'title': text_format.format_title(ui_node.text),
-            'type': 'value',
-            'children': [],
-            'count': 1,
-            'search_count': search_count,
-        })
-        root['count'] += 1
-        root['search_count'] += search_count
+        for sv in stat_vars[pos.key]:
+            if pos.cpv == sv.pv:
+                root['children'].append({
+                    'populationType': ui_node.pop_type,
```

Comment:
don't need the populationType field now

Reply:
done

Reply:
I kept populationType, because it is used in the filter_and_sort functions in the text_format.py, and also used for rearranging some nodes.